### ISSUE TITLE
Creating new key signature is not accessible

### ISSUE THREAD

**Issue by piyanistumutkorkmaz** [musescore/MuseScore#25631]
### Your idea

I want to create accessible new Key signature with quarter sounds for Turkish Music and Arabic Music. Since I am blind, I cannot currently access the relevant window with the keyboard.

### Problem to be solved

In the main palette window, I will be able to create a custom key signature by accessing the Key signature window with the keyboard.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by jmwmusic** [comment]
Is this something that sighted users can currently do in MuseScore?  If so and it is not accessible then this is a bug.  Can you give specific steps of how to do this and where it does not work with a screen reader/keyboard only input?

**Comment by piyanistumutkorkmaz** [comment]
Hello,When we open the main pallet window with Shift+F9:The new key signature is created under the Branch of Key Signatureres.I could not manage to create it with nvda.
________________________________
Gönderen: jmwmusic ***@***.***>
Gönderildi: 20 Mayıs 2025 Salı 10:46
Kime: musescore/MuseScore ***@***.***>
Bilgi: piyanistumutkorkmaz ***@***.***>; Author ***@***.***>
Konu: Re: [musescore/MuseScore] Creating new key signature is not accessible (Issue #25631)

[https://avatars.githubusercontent.com/u/145303606?s=20&v=4]jmwmusic left a comment (musescore/MuseScore#25631)<https://github.com/musescore/MuseScore/issues/25631#issuecomment-2893310500>

Is this something that sighted users can currently do in MuseScore? If so and it is not accessible then this is a bug. Can you give specific steps of how to do this and where it does not work with a screen reader/keyboard only input?

**Comment by MarcSabatella** [comment]
Yes, sighted users can create custom key signatures.  It's one of the few
remaining things that specifically requires drag & drop.

Marc
